keel-core/keel-core.gradle.kts

```diff
@@ -47,6 +47,5 @@ dependencies {
   testImplementation("dev.minutest:minutest")
 
   testImplementation("org.assertj:assertj-core")
-  testImplementation("org.jetbrains.kotlinx:kotlinx-coroutines-test")
```

Comment:
Because `runBlockingTest` is no longer being used, don't need this anymore.

Reply:
good riddance